report/src/main/webapp/app/components/keyboardShortcuts.directive.js

```diff
@@ -141,36 +142,41 @@ define(['angularAMD', 'userSettingsService'], function (angularAMD) {
             if (!_.isEmpty($(suiteContainer).nextAll(
                     '.aside-report:not(.is-hidden)').first().find(
                     '.test-name'))) {
+              currentTab = $('.nav-tabs > .nav-item').filter('.active').text().replace(/\s/g, '');
```

Comment:
is it possible to have more than one `.nav-item` element with class `active`? if not, why not going without filtering?

Reply:
after discussion I'm ok with keeping it as it is